app/src/main/assets/style.css

```diff
@@ -0,0 +1,9 @@
+body, html {
+    background: #221F1F;
+    color: #E1E5EA;
+    font: -apple-system-body;
```

Comment:
I guess this doesn't have any affect on Android. As far as I understand it this will use the font of the system to render the web content. Is there something like that on Android? Otherwise we could try to use Roboto or just not specify any font at all.